Development/BIDS/BIDS_createTemplatesImportConfig.m

```diff
@@ -118,6 +118,7 @@
 for iFlavor=1:length(listOfConfigFlavors)
     fprintf('%s\n', listOfConfigFlavors{iFlavor});
     imPar = ExploreASL_ImportConfig(listOfConfigFlavors{iFlavor});
+    imPar.TempRoot = imPar.AnalysisRoot;
```

Comment:
if the camelCase of imPar starts with a small cap, should tempRoot and analysisRoot also start with a small cap as well?

Also, should we not change AnalysisRoot throughout to be TempRoot?

Reply:
AnalysisRoot started with a capital A and the other path descriptions there, too, so I thought it would be more consistent this way.
I just kept AnalysisRoot in the configFiles for backwards compatibility. We don't need it anymore, but in case someone would use it with some old script, it would probably be better to keep the field definition in the old json files. They will be removed sooner or later anyway, so let's just ignore this.